docker-compose.yaml

```diff
@@ -440,6 +440,20 @@ services:
     ports:
       - "8080:8080"
       - "9990:9990"
+    volumes:
+      # mount volumes for JaCoCo test coverage for our integration tests
+      # see: https://blog.akquinet.de/2018/09/06/test-coverage-for-containerized-java-apps/
+      - ./build/jacoco/itest/jacoco-agent:/jacoco-agent
+      - ./build/jacoco/itest/jacoco-report:/jacoco-report
+    # override entrypoint to enable the JaCoCo agent to generate test coverage for our integration tests
+    entrypoint: [
+      "java",
+      "-javaagent:/jacoco-agent/org.jacoco.agent-runtime.jar=destfile=/jacoco-report/jacoco-it.exec",
```

Comment:
Hmm, this is now breaking our local Docker Compose set up when you have not run `gradlew copyJacocoAgentForItest` beforehand.. Would be nice if I could make this entrypoint optional somehow.

Reply:
I fixed this now by using an env var to override this. Not super pretty but I think good enough.